### PR TITLE
Remove unused code related to Che

### PR DESCRIPTION
**Pull request by baijum**
Ref. https://issues.redhat.com/browse/SANDBOX-345

### CODE REVIEW

File: pkg/configuration/load_test.go [hunk 1]
```diff
@@ -218,59 +218,59 @@ func TestNoWatchNamespaceSetWhenLoadingConfigMap(t *testing.T) {
 
 func TestLoadSecrets(t *testing.T) {
 	secretData := map[string][]byte{
-		"che-admin-username": []byte("cheadmin"),
-		"che-admin-password": []byte("password"),
+		"admin-username": []byte("cheadmin"),
+		"admin-password": []byte("password"),
```
Comment (alexeykazakov) [inline pkg/configuration/load_test.go:222]: Those test secrets values are still che related. Let's maybe change them to something more generic. Something like:
```suggestion
		"key-1": []byte("value-1"),
		"key-2": []byte("value-2"),
```

Comment (rajivnathan) [reply pkg/configuration/load_test.go:222]: +1

Comment (baijum) [reply pkg/configuration/load_test.go]: Done. c5c1e6b3ad0f2fe6731f35f2cebc4a8b9288f5a9

**Review by alexeykazakov — APPROVED** [review]
Looks good. Thank you.

**Pull request merged**